### PR TITLE
iDNES.cz ruleset

### PR DESCRIPTION
**Pull request by MikkCZ**

### CODE REVIEW

**Comment by MikkCZ** [comment]
@fuglede r?

**Comment by MikkCZ** [comment]
@J0WI , could you please look at this ruleset too?

**Comment by J0WI** [comment]
This rule has several issues:
- The ruleset name is rajce.net, but the targets belongs to rajce.idnes.cz.
- The ruleset contains a wildcard, but has no test urls.
- rajce.idnes.cz itdelf is already a subdomain of idnes.cz.

It's recommend to [search](https://github.com/EFForg/https-everywhere/blob/master/ruleset-style.md) for subdomains in *.idnes.cz and list all the targets. Ifthe host relly support HTTPS on all subdomains, I wildcard target is possible but requires a list of test urls.

**Comment by MikkCZ** [comment]
Thank you for the comment. I will address them in the next commit.
- rajce.net is the old domain and still the website name, I will probably rename it to idnes.cz completely, despite idnes.cz web is not on HTTPS yet
- the wildcard is a must here, because rajce create a subdomain for every user, like https://jenamalinach.rajce.idnes.cz/, I will add some test subdomains

**Comment by MikkCZ** [comment]
Updated to cover all idnes.cz subdomains I have found and work with HTTPS.

File: src/chrome/content/rules/idnes-cz.xml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+	<target host="revue.idnes.cz" />
+	<target host="rungo.idnes.cz" />
+	<target host="sdeleni.idnes.cz" />
+	<target host="servis.idnes.cz" />
+	<target host="sport.idnes.cz" />
+	<target host="technet.idnes.cz" />
+	<target host="tvprogram.idnes.cz" />
+	<target host="video.idnes.cz" />
+	<target host="wiki.idnes.cz" />
+	<target host="zpravy.idnes.cz" />
```
Comment (J0WI) [inline src/chrome/content/rules/idnes-cz.xml]: I found some more:

idnes.cz
www.idnes.cz
alik.idnes.cz
blog.idnes.cz
brno.idnes.cz
budejovice.idnes.cz
cestiny.idnes.cz
data.idnes.cz
fotbal.idnes.cz
hledej.idnes.cz
hokej.idnes.cz
jihlava.idnes.cz
kultura.zpravy.idnes.cz
liberec.idnes.cz
mobilnihry.idnes.cz
olomouc.idnes.cz
onlinehry.idnes.cz
ostrava.idnes.cz
pardubice.idnes.cz
plzen.idnes.cz
praha.idnes.cz
r.idnes.cz
reality.idnes.cz
rss.idnes.cz
servix.idnes.cz
topkontakt.idnes.cz
usti.idnes.cz
vary.idnes.cz
vice.idnes.cz
volby.idnes.cz
xman.idnes.cz
zlin.idnes.cz
kultura.zpravy.idnes.cz


Please add non-working ones as comments on the top, to document the issue.

**Comment by MikkCZ** [comment]
Sorry @J0WI it took me so long. No I have added all the subdomains you have detected and manually tested them.

**Comment by deepj** [comment]
Please, revert this PR. After this, each subdomain of idnes.cz is broken. It

Try https://zpravy.idnes.cz/ or https://kraje.idnes.cz/ or https://technet.idnes.cz/

**Comment by MikkCZ** [comment]
There is already an issue https://github.com/EFForg/https-everywhere/issues/7903 and PR for it https://github.com/EFForg/https-everywhere/pull/7941. It used to work OK, but seems iDNES decided to block HTTPS instead of making it work by default.

**Pull request merged**